### ISSUE TITLE
Check parameters and update if required: nu and upsilon

### ISSUE THREAD

**Issue by grinnellm** [grinnellm/SpawnIndex#11]
Working on this with Jake.

**Comment by grinnellm** [comment]
Changed the R package and Tech Report so that `\nu` is now the proportion of SOK product that is kelp. Used to be the proportion that was eggs, not kelp. Now the equation has a term `1 - \nu` instead of `\nu` to reflect this change. We found the location of the 12% reference in Shields et al. (1985) in the second last paragraph of page 13. This seems to include data from 1982 that is not shown in Table 10. And our "uncertainty table" indicates the SD for this value (4.2).
Did a similar change for `\upsilon` -- now it's the proportion weight increase due to brining. The equation now has the term `1 / (1 + \upsilon)` to reflect this change. We are still working on quantifying this value from Whyte and Englar (1977).
